retail-forecast/requirements.txt

```diff
@@ -0,0 +1,9 @@
+zenml~=0.82.0
+numpy>=1.20.0
+pandas>=1.3.0
+matplotlib>=3.5.0
+prophet>=1.1.0
+typing_extensions>=4.0.0
```

Comment:
presumably also less than the next major version, too? esp things like numpy + pandas which often have quite big breaking changes on their big major releases

Reply:
No i think this works as it is on the latest ones

Reply:
Yeah but wouldn't it break when numpy releases 2.x and pandas release 2.x etc?

Reply:
numpy already has 2.x and pandas too